### ISSUE TITLE
Crashes with java.lang.NullPointerException on Android

### ISSUE THREAD

**Issue by rsnay** [voximplant/react-native-voximplant#40]
Hi!

My team recently published an app on the Google Play store that uses react-native-voximplant. Integration was very smooth, and we're pleased with how quickly we could get going with it.

Performance is stable and reliable, but we have had several instances of the following crash be reported. Here is one of the reports:

```
Jun 6, 11:06 AM on app version 4
Samsung Galaxy S7 Edge (hero2qlteatt), 4096MB RAM, Android 7.0

java.lang.NullPointerException: 
  at com.zingaya.voximplant.VoxImplantClientImp$10.run (VoxImplantClientImp.java:518)
  at java.util.TimerThread.mainLoop (Timer.java:555)
  at java.util.TimerThread.run (Timer.java:505)
```

Has anyone else experienced a crash from this? If so, is it caused by the underlying voximplant library, or the code I write in react native?

Thanks!

**Comment by YuliaGrigorieva** [comment]
Hello!

Thank you for reporting this issue!

Could you please indicate the React Native Voximplant SDK version that you are using?

This issue has never been reported, however we will investigate it and will improve the stability of the the code.
You can also collect the Voximplant logs (even if the issue is not reproducible on your side) and send them to support@voximplant.com, so we can check the Voximplant SDK api sequence calls and give you some recommendations if required.

Best regards,
Yulia Grigorieva

**Comment by rsnay** [comment]
Hi Yulia,

Thanks for the response! We're using version 0.2.1 of the Voximplant React Native SDK.

I don't have enough information right now to determine which calls/logs are associated with these crashes, but I will send the logs your way if I can.

Thanks again!

**Comment by YuliaGrigorieva** [comment]
Hello!

We have just published react-native-voximplant 0.2.2 that contains changes to avoid this crash. 
Please update the sdk and monitor the issue.

I'm closing the issue now, but feel free to reopen it in case if the issue is reproduced.

Best regards,
Yulia Grigorieva
